src/components/Team/index.tsx

```diff
@@ -102,26 +107,18 @@ const TEAM_MEMBERS: TeamMember[] = [
   {
     avatar: '/avatars/fer.webp',
     name: 'Fernando Ramirez',
-    role: 'Frontend Developer',
+    role: 'Business Developer, Web3 Developer',
```

Comment:
@FedeMadoery talking with @Agupane we agreed on 'Web3 Developer & Business Relations', it could be also 'Web3 and Business Developer'. On any case I would mention the technical role first, because is what I do the most, and additionally separating with comma and repeating 'Developer' is not very nice I think